.github/workflows/fqdn-perf.yaml

```diff
@@ -52,6 +52,7 @@ concurrency:
     ${{ github.workflow }}
     ${{ github.event_name }}
     ${{
+      (github.event_name == 'push' && github.sha) ||
```

Comment:
This test should not be triggered on renovate push, as it is not part of the default Ariane suite (i.e., /test)

Reply:
oh, good catch